log4j-1.2-api/src/main/java/org/apache/log4j/xml/XmlConfiguration.java

```diff
@@ -675,24 +675,13 @@ private void parseLevel(Element element, LoggerConfig logger, boolean isRoot) {
             }
         } else {
             String className = subst(element.getAttribute(CLASS_ATTR));
+            final Level level;
             if (EMPTY_STR.equals(className)) {
-                logger.setLevel(OptionConverter.convertLevel(priStr, org.apache.logging.log4j.Level.DEBUG));
+                level = OptionConverter.toLevel(priStr, Level.DEBUG);
```

Comment:
THere are several places where we specify a default level (Level.DEBUG) in this method and likely elsewhere. It would be nicer to have a default level defined in a constant IMO; this would make it obvious that this is a "default" as opposed to "in this case, we have level Foo". WDYT?

Reply:
I'll look into that. There are actually two levels used throughout the code: `Level.DEBUG` in places where we expect an operation to fail and `Level.ERROR` where a conversion failure is unexpected (like in the `convertLevel` methods).